app/views/invoices/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with controller: :invoices %>
+<%= link_to 'Create new invoice', new_invoice_path, remote: true %>
+
+<h2>Your invoices</h2>
+<% @invoices.each do |invoice| %>
+  <ul>
+    Invoice number: <%= invoice.id %>
+    Invoice date: <%= invoice.date %>
```

Comment:
I would use the createt_at time stamp for this

Reply:
Yeeah that was my initial thought but I was thinking that it might be, technically, legal to start a draft of an invoice at a date that is before the invoicing date so you could possibly want to change the date. Though I'm not totally sure about that.